spec.bs

```diff
@@ -436,20 +440,22 @@ Each {{URLPattern}} object has an associated <dfn for=URLPattern>hash component<
 <div algorithm>
   The <dfn method for="URLPattern">test(|input|, |baseURL|)</dfn> method steps are:
 
-  1. Let |result| be the result of [=URLPattern/match=] given [=this=], |input|, and |baseURL| if given.
+  1. Let |result| be the result of [=URLPattern/match=] given [=this=]'s associated [=URL Pattern=], |input|, and |baseURL| if given.
   1. If |result| is null, return false.
   1. Return true.
 </div>
 
 <div algorithm>
   The <dfn method for="URLPattern">exec(|input|, |baseURL|)</dfn> method steps are:
 
-  1. Return the result of [=URLPattern/match=] given [=this=], |input|, and |baseURL| if given.
+  1. Return the result of [=URLPattern/match=] given [=this=]'s associated [=URL Pattern=], |input|, and |baseURL| if given.
 </div>
 
 <h3 id=urlpattern-internals>Internals</h3>
 
-A {{URLPattern}} is associated with multiple <dfn>component</dfn> [=structs=].
+A {{URLPattern}} has an associated <dfn>URL pattern</dfn> [=structs=].
```

Comment:
You need separate definitions for the type and the field.

Something like:

```html
A <dfn>URL pattern</dfn> is a [=struct=] with the following [=struct/items=]:

* A <dfn for="URL pattern">protocol component</dfn>, a [=component=]
* ...
```

and then in the definition of `{{URLPattern}}`:

```html
Each {{URLPattern}} has a <dfn for="URLPattern">URL pattern</dfn>, a [=URL pattern=].
```

If you find "URL pattern" to be too confusing of a name for this field, good alternatives would be "backing struct", "pattern", or "associated URL pattern". Different specs use different styles so it's up to you/the editors.

Reply:
done.